unstructured_inference/inference/layout.py

```diff
@@ -156,3 +164,11 @@ def process_file_with_model(filename: str, model_name: str) -> DocumentLayout:
     model = None if model_name is None else get_model(model_name)
     layout = DocumentLayout.from_file(filename, model=model)
     return layout
+
+
+def cid_ratio(text: str) -> float:
+    """Gets ratio of unknown 'cid' characters extracted from text to all characters."""
```

Comment:
This feels not super cheap computationally, but I don't see a great way around it.
For minor optimization, could do:
```
if len(text) < 6:
   return 1.0
```

Reply:
actually, one could do plain old string.find for a first match of `(cid:` and return 1.0 if not found, otherwise the regex party.

Reply:
Added some pre-checks before running regex as suggested.